packages/react/src/runtime/shorthand.ts

```diff
@@ -0,0 +1,108 @@
+import type { ShorthandProperties } from '@compiled/utils';
+
+const shorthandFor: Record<ShorthandProperties, true> = {
```

Comment:
Could change to `1` instead of `true`, but I'd expect compilation might do this anyways.

I like your approach because types protects it 👌

Reply:
ah that's a good idea

it doesn't look like the current compilation does that, so changing `true` to 1 would definitely help with byte size

i plan to make another commit to make overlapping properties with the same "depth" (e.g. `borderTop` and `borderColor`) deterministically sorted too, so this file will change some more 😅